less/release-9.less

```diff
@@ -0,0 +1,351 @@
+.flex-wrap {
+  flex-wrap: wrap;
+}
+
+.jumbotron-wrapper {
+  overflow: hidden;
+  position: relative;
+}
+
+.hide {
```

Comment:
Be careful before introducing new classes since they might already exist:
https://getbootstrap.com/docs/3.4/css/#helper-classes-show-hide